src/tv2-common/videoSwitchers/__tests__/AtemToTricasterDveConverter.spec.ts

```diff
@@ -0,0 +1,199 @@
+import { TSR } from '../../../../../tv-automation-server-core/packages/blueprints-integration/src'
```

Comment:
Seems like a path that needs a quick refactor.

Reply:
Yes, I was looking for this one. It's causing some issues...